src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/commands.py

```diff
@@ -20,6 +20,10 @@
 
 # VM
 
+def empty_on_404(ex):
+    if ex.status_code == 404:
```

Comment:
Question:
For `az vm show -g myrg -n vmdoesntexist` should output be `` (empty string) or `{}` (empty dict)?

Currently, it would be empty string.